### PR TITLE
Changed type of the logged indentation depth to an enum type

### LINKED ISSUE #343: Indentation depth as enum
The PerformanceLoggingService logs among other things the indentation depth of a method. This indentation depth is a number (starting from 0) and has so far also been stated as a number. However, the indentation depth is to be mapped as an enum in the PerformanceLoggingService (with number as value).

### PR DESCRIPTION
**Pull request by DanielSteger**

### CODE REVIEW

File: grails-app/services/de/iteratec/osm/measurement/schedule/JobRunService.groovy [hunk 1]
```diff
@@ -95,10 +97,10 @@ class JobRunService {
         JobResultStatus jobResultStatus = JobResultStatus.PERSISTANCE_ERROR
         try {
             WptResultXml resultXml
-            performanceLoggingService.logExecutionTime(DEBUG, "Polling jobrun ${testId} of job ${job.id}: fetching results from wptrserver.", 1) {
+            performanceLoggingService.logExecutionTime(DEBUG, "Polling jobrun ${testId} of job ${job.id}: fetching results from wptrserver.", IndentationDepth.ONE) {
                 resultXml = wptInstructionService.fetchResult(job.location.wptServer, testId)
             }
-            performanceLoggingService.logExecutionTime(DEBUG, "Handling WPT result of jobrun ${testId} of job ${job.id}", 1) {
+            performanceLoggingService.logExecutionTime(DEBUG, "Handling WPT result of jobrun ${testId} of job ${job.id}", IndentationDepth.ONE) {
```
Comment (BennyLi) [inline grails-app/services/de/iteratec/osm/measurement/schedule/JobRunService.groovy:103]: This is much easier to read and use.

File: grails-app/services/de/iteratec/osm/measurement/environment/wptserver/WptInstructionService.groovy [hunk 2]
```diff
@@ -89,7 +89,7 @@ class WptInstructionService {
         Map paramsNotNull = parameters.findAll { k, v -> v != null }
         def result = null
         log.debug("Try to launch job at wptserver=${wptServer} with params=${parameters}")
-        performanceLoggingService.logExecutionTime(DEBUG, "Launching job ${job.label}: Calling initial runtest on wptserver.", 1) {
+        performanceLoggingService.logExecutionTime(DEBUG, "Launching job ${job.label}: Calling initial runtest on wptserver.", PerformanceLoggingService.IndentationDepth.ONE) {
```
Comment (BennyLi) [inline grails-app/services/de/iteratec/osm/measurement/environment/wptserver/WptInstructionService.groovy]: Couldn't you remove the `PerformanceLoggingService.` part?

**Review by BennyLi — CHANGES_REQUESTED** [review]

File: grails-app/services/de/iteratec/osm/measurement/environment/wptserver/WptInstructionService.groovy (hunk 2 above)
Comment (BennyLi) [reply grails-app/services/de/iteratec/osm/measurement/environment/wptserver/WptInstructionService.groovy]: This occurs in multiple files.

Comment (DanielSteger) [reply grails-app/services/de/iteratec/osm/measurement/environment/wptserver/WptInstructionService.groovy]: I'll change it.

**Review by BennyLi — APPROVED** [review]

**Pull request merged**